routes/web.php

```diff
@@ -79,6 +79,7 @@
 Route::post('/api/Settings::getAll', 'SettingsController@getAll')->middleware('admin');
 Route::post('/api/Settings::saveAll', 'SettingsController@saveAll')->middleware('admin');
 Route::post('/api/Settings::setOverlayType', 'SettingsController@setImageOverlayType')->middleware('admin');
+Route::post('/api/Settings::setDropboxKey', 'SettingsController@setDropboxKey')->middleware('admin');
```

Comment:
Does that mean that setting the Dropbox key didn't work at all in v4 until now? :confused:

Reply:
yup :laughing: